requirements.txt

```diff
@@ -19,7 +19,7 @@
 # under the License.
 #
 
-## To install all dependencies, run: pip3 install -r requirements.txt
+## To install all dependencies, run: pip install -r requirements.txt
```

Comment:
`pip3 install` was necessary particularly on Mac when the user had both Python2 and Python3 installed, but this setup is becoming less common.